src/main/java/jalau/at18/azul/CenterTileBoard.java

```diff
@@ -0,0 +1,30 @@
+package jalau.at18.azul;
+import java.util.*;
+
+public class CenterTileBoard extends ArrayList<Character> {
+
+
+    public CenterTileBoard() {
+        super();
+        this.add('F');
+    }
+
+    public CenterTileBoard removeTilesOfSpecificColor(char color) {
```

Comment:
This might be already implemented, since you are inheriting from ArrayList, this class already has a method to remove an specific element: https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/util/ArrayList.html#remove(java.lang.Object)

Try to use that one, and even better, if you use `Tile` instead of `Character` like in my previous comment, you don't need to implement this logic, it already exists 😄

Reply:
I have gotten rid of the for and only use the removeAll